manual/manual/library/stdlib.etex

```diff
@@ -8,7 +8,7 @@ having to add any ".cmo" file on the command line for the linking
 phase. Similarly, in interactive use, these globals can be used in
 toplevel phrases without having to load any ".cmo" file in memory.
 
-Unlike the "Pervasives" module from the core library, the modules from the
+Unlike the "Stdlib" module from the core library, the modules from the
```

Comment:
This sentence seems a bit confusing since it refers to Stdlib in the two opposite propositions. Maybe:
 
> Unlike the core "Stdlib" module, those submodules are not automatically ``opened'' when a compilation starts,

Reply:
I pushed a new commit, but apparently it's 1991 and we can't have clear colour in GitHub comments, so here's a screenshot of a better diff tool...

![image](https://user-images.githubusercontent.com/5250680/59998872-d0770a80-9660-11e9-8d56-9d5452c0443e.png)

Reply:
This seems better, thanks!